src/core/src/OmniPolygonRasterOverlay.cpp

```diff
@@ -16,7 +16,8 @@
 namespace cesium::omniverse {
 
 OmniPolygonRasterOverlay::OmniPolygonRasterOverlay(Context* pContext, const pxr::SdfPath& path)
-    : OmniRasterOverlay(pContext, path) {}
+    : OmniRasterOverlay(pContext, path)
+    , _pExcluder(nullptr) {}
```

Comment:
`shared_ptr` are default constructed as nullptr, so `_pExcluder(nullptr)` isn't necessary.

It would be a different story if `_pExcluder` were a raw pointer.